.github/workflows/.windows.yml

```diff
@@ -12,6 +12,7 @@ name: .windows
 permissions:
   contents: read
 
+
```

Comment:
Intentional double blank line? :eyes:

Reply:
oh! very intentionally not intentional.

Oh! I know what happened; I first tried if it would accept a top-level timeout-minutes (similar to permissions), because that would make the most logical place to put a default.

But unfortunately it doesn't, so then I picked "after every `runs-on` as a conventions" (closest to "top level").

Reply:
There you go:

- https://github.com/moby/moby/pull/48636